pages/e-lab.tsx

```diff
@@ -40,8 +44,10 @@ function Hero() {
               technology
             </p>
 
-            <Button className="border-none bg-gradient-to-b from-yellow-500 to-red-500">
-              Apply now
+            <Button className="asChild border-none bg-gradient-to-b from-yellow-500 to-red-500">
```

Comment:
```suggestion
            <Button className="border-none bg-gradient-to-b from-yellow-500 to-red-500" asChild>
```

asChild is a prop from [radix-ui/slot](https://www.radix-ui.com/docs/primitives/utilities/slot) for polymorphic components. Since the Button class is purely for visual purposes we inject its props into its child with `asChild`

Take a look at the implementation of the Button component if you want to find out more